bftengine/src/bftengine/ViewsManager.hpp

```diff
@@ -145,9 +145,12 @@ class ViewsManager {
 
   void storeComplaintForHigherView(std::unique_ptr<ReplicaAsksToLeaveViewMsg> &&complaintMessage);
   bool hasQuorumToJumpToHigherView() const { return complainedReplicasForHigherView.hasQuorumToLeaveView(); }
```

Comment:
One remark that is related to the previous PR with refactors in ViewsManager that I missed then, but I think it is not too late to make it better :) Let's make those 2 functions protected, we only use them when processing View Change Messages when we iterate the complaints they contain.